src/odemis/util/conversion.py

```diff
@@ -357,12 +357,12 @@ def get_tile_md_pos(i, tile_size, tileda, origda):
     tmat = get_img_transformation_matrix(md)
 
     # Converts the tile_rel_to_img_center_pixels array of coordinates to a 2 x 1 matrix
-    # The numpy.matrix(array) function returns a 1 x 2 matrix, so .getT() is called
+    # The numpy.array(array) function returns a 1 x 2 matrix, so .getT() is called
```

Comment:
you forgot to update "so .getT() is called", should now be "so .transpose() is called"